introduction.md

```diff
@@ -0,0 +1 @@
+This is my introduction document as I was told to create it by git mastery on code school.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.